### PR TITLE
[no-issue]: Convert database tests to jest

### PR DESCRIPTION
**Pull request by kael89**
2 more packages (`central-server`, `web-config-server`) and then `mocha` can go! :tada: 

Most of the job was done using a community [codemod](https://jestjs.io/docs/migration-guide#jest-codemods)

### CODE REVIEW

File: packages/database/src/__tests__/modelClasses/DataElement.test.js [hunk 1]
```diff
@@ -101,17 +99,5 @@ describe('DataElement', () => {
         );
       });
     });
-
-    it('should remove valid fields when empty', () => {
```
Comment (kael89) [inline packages/database/src/__tests__/modelClasses/DataElement.test.js:105]: This was a duplicate test case

**Review by edmofro — APPROVED** [review]
Looked through and all looks good to me! Great that you caught a few things like that dupe test, some unneeded awaits, and even the language used to describe each test case.

Thanks for swooping in with your tech debt cleaning super powers, Open Source Community 😁

File: packages/database/src/__tests__/utilities/migration.test.js [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { removeArrayValue, replaceArrayValue, updateValues } from '../../utilities/migration';
+
+const dbStub = {
+  runSql: (query, replacementParams) => ({ query, replacementParams }),
+};
+
+describe('migrationUtilities', () => {
```
Comment (kael89) [inline packages/database/src/__tests__/utilities/migration.test.js:12]: Restructure this file a bit to make it less verbose

**Comment by edmofro** [comment]
Have enabled auto-merge, will be committed to dev as soon as CI/CD checks have passed.

**Comment by kael89** [comment]
> Have enabled auto-merge, will be committed to dev as soon as CI/CD checks have passed.

Ah nice, that's a handy feature! A few dev scripts have been touched because of the test folder update. I guess that if anything has been messed up the CI/CD will tell :-)

File: packages/database/src/__tests__/changeHandlers/SurveyResponseOutdater.test.js [hunk 3]
```diff
@@ -76,18 +76,19 @@ describe('SurveyResponseOutdater', () => {
...
+      // const responseDescriptionFields = {
+      //   survey_code: survey.code,
+      //   entity_id: entity.code,
+      //   data_time: surveyResponse.data_time,
+      // };
+      // const message = `Failed assertion for survey response ${JSON.stringify(
+      //   responseDescriptionFields,
+      // )}`;
+
+      expect(surveyResponse).toHaveProperty('outdated', expected);
```
Comment (kael89) [inline packages/database/src/__tests__/changeHandlers/SurveyResponseOutdater.test.js:91]: Using a 3rd parameter (for a custom message) is not only unsupported by `jest`, but it causes `toHaveProperty` to always pass! :fearful:

Interestingly this is not the case with `toBe`, which fails as expected. I double-checked that the only instance where custom messages are still used in this package is in [AnalyticsRefresher.test.js](https://github.com/beyondessential/tupaia/pull/4188/files#diff-ae6df06af6401ae30c75e21dac78e71ba688d5`7de5edf2001266e22f8e3b4f8e), for `.toBe` assertions.

**Pull request merged**